lib/client.js

```diff
@@ -354,38 +354,33 @@ class Client extends EventEmitter {
   }
 
   close (cb) {
-    if (this.closed) {
-      return
```

Comment:
I think those two lines might still be needed somewhere?

Reply:
No, not required. If it's already closed it doesn't do anything.